sale_partcode_substitution/wizard/sale_partcode_replacement.py

```diff
@@ -53,5 +52,7 @@ def change_products_partcode(self):
                         )
                         vals = self._finalize_vals(vals, line, new_part)
                         line.write(vals)
-        sale._amount_all()  # TODO: Why is this needed now?
```

Comment:
Just don't change this line at all. By changing it you are rewrting its blame history.